### ISSUE TITLE
Daemon crashes when detaching from project with anonymous platform

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#490]
**Reported by Ageless on 3 Jun 37920966 22:13 UTC**
Tested on BOINC 5.10.30, project Enigma@Home. 
Using the optimized application there with the anonymous platform file, app_info.xml[[br]]
Detaching from project will crash the BOINC daemon. 
```
Event Type:	Error
Event Source:	Service Control Manager
Event Category:	None
Event ID:	7031
Date:		02-Dec-07
Time:		15:55:48
User:		N/A
Computer:	XXXXXXXXXXXXX
Description:
The BOINC service terminated unexpectedly.  It has done this 1 time(s).  
The following corrective action will be taken in 0 milliseconds: No action. 
```

The service/daemon should keep on running.

**Comment by romw** [comment]
**Commented by Nicolas on 29 Jan 37922526 16:53 UTC**
Anything interesting on stdoutdae or stderrdae?

**Comment by romw** [comment]
**Commented by Ageless on 29 Aug 37923145 00:00 UTC**
Nothing of use in stdoutdae.txt:[15:55:46 [Enigma@Home]([br]]
02-Dec-2007) Resetting project
02-Dec-2007 15:55:46 [Enigma@Home] Detaching from project

stderrdae.txt had a crash dump in it though. I've attached that one.

**Comment by romw** [comment]
**Commented by davea on 21 Feb 37924009 22:40 UTC**
(In [14348])     - client: fix bug that would falsely report DLL init errors
    - client: don't crash when detach a project using anonymous platform
        (fixes #511)
